vendor/github.com/nginx/agent/sdk/v2/proto/host.proto

```diff
@@ -91,7 +91,7 @@ message CpuInfo {
   int32 cpus = 6 [(gogoproto.jsontag) = "cpus"];
   // Type of hypervisor (e.g guest or host)
   string virtualization = 7 [(gogoproto.jsontag) = "virtualization"];
-  // Map of caches with names as the keys and size in bytes as the values
+  // Map of caches with names as the keys and size in formatted bytes as the values
```

Comment:
I'm late to the design here, but is the agent the right place for this formatting? This is a breaking API change if other consumers; they might have code like:

```go
for name, value := range cache {
  bytes, err := strconv.ParseInt(value, 10, 64)
  if err != nil {
    panic("bad data from agent")
  }
  // ...
}
```

That code would work before this change, when the cache values were things like "131072", but will choke on "128 KiB".

Are we confident that agent consumers want the formatted version?

Reply:
I had this formatted bytes logic in place as per our existing amplify agent logic as here https://github.com/nginxinc/nginx-amplify-agent/blob/fa4fb352ee6db134899364f5c8a3e0f1bd536ed3/amplify/agent/collectors/system/meta.py#L109